### PR TITLE
DOC-11535 Feedback on Crash Logs | Couchbase Docs

### PR DESCRIPTION
**Pull request by RayOffiah**
https://issues.couchbase.com/browse/DOC-11535

### CODE REVIEW

File: modules/ROOT/pages/_partials/symbolicate-ios.adoc [hunk 1]
```diff
@@ -0,0 +1,79 @@
+== Symbolication
+
+In order to make sense of a crash report you will need to symbolicate
+ its content.
+Symbolication being the process of replacing memory addresses in a crash report with human-readable function names and line numbers.
+To do this you need to have debug symbol (`.dSYM`) files for the application and for Couchbase Lite's framework.
+
+== Finding local dSYM
```
Comment (pasin) [inline modules/ROOT/pages/_partials/symbolicate-ios.adoc:7]: I don't think this section is needed as it's for the application dSYM file (not CouchbaseLite dSYM file).

File: modules/ROOT/pages/_partials/symbolicate-ios.adoc [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+.dSYM in CouchbaseLiteSwift
+====
+----
+couchbase-lite-swift_enterprise_2.7.0/iOS/CouchbaseLiteSwift.framework.dSYM
+----
+====
+
+== Bitcode enabled Apps
+
+Apple creates new dSYM files for apps uploaded to the _App Store_ or _TestFlight_ with _Bitcode_ enabled.
```
Comment (pasin) [inline modules/ROOT/pages/_partials/symbolicate-ios.adoc]: I understand that Apple doesn't include bitcode to the app anymore, but how the developers can get the dSYM file from the archived app or from the App Store is still valid. So I think this section can be rewritten such as "How to get the dSYM file from the app."

File: modules/ROOT/pages/_partials/symbolicate-ios.adoc [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+* Firebase Crashlytics: https://firebase.google.com/docs/crashlytics/get-deobfuscated-reports
+* Instabug: https://docs.instabug.com/docs/ios-symbolication
+* Bugsee: https://docs.bugsee.com/sdk/ios/symbolication/
+* Microsoft Visual Studio App Center: https://docs.microsoft.com/en-us/appcenter/diagnostics/ios-symbolication
+* Apteligent: https://www.apteligent.com/technical-resource/symbolicating-an-ios-crash-report/
+====
+
+== Obfuscated symbols (`_hidden`)
+
+Xcode includes obfuscated symbols in dSYM files if apps with _Bitcode_ enabled are uploaded to Apple without symbols.
```
Comment (pasin) [inline modules/ROOT/pages/_partials/symbolicate-ios.adoc]: Remove "if apps with _Bitcode_ enabled".

File: modules/ROOT/pages/_partials/symbolicate-jvm.adoc [hunk 4]
```diff
@@ -0,0 +1,17 @@
+// BEGIN -- inclusion -- {root-partials}symbolicate-jvm.adoc
```
Comment (pasin) [inline modules/ROOT/pages/_partials/symbolicate-jvm.adoc]: I don't think this section is applicable for Java. Java / Android will have their own way for symbolicating the symbols. Please check with @bmeike.

File: modules/ROOT/pages/_partials/symbolicate-net.adoc [hunk 5]
```diff
@@ -0,0 +1,17 @@
+// BEGIN -- inclusion -- {root-partials}symbolicate-net.adoc
```
Comment (pasin) [inline modules/ROOT/pages/_partials/symbolicate-net.adoc]: I don't think this section is applicable for .NET. .NET will have their own way for symbolicating the symbols. Please check with @borrrden .

**Review by pasin — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/_partials/symbolicate-jvm.adoc (hunk 4 above)
Comment (RayOffiah) [reply modules/ROOT/pages/_partials/symbolicate-jvm.adoc]: Turns out that this file didn't have a menu item, so I think it was deleted on purpose. I've removed the file.

File: modules/ROOT/pages/_partials/symbolicate-net.adoc (hunk 5 above)
Comment (RayOffiah) [reply modules/ROOT/pages/_partials/symbolicate-net.adoc]: Turns out that this file didn't have a menu item, so I think it was deleted on purpose. I've removed the file.

**Pull request merged**